### ISSUE TITLE
Error on sails lift

### ISSUE THREAD

**Issue by iit-federico-bozzini** [OpenServicesEU/waterlock-ldap-auth#1]
I configured the waterlock.js file according to the readme

```
authMethod: [
        {
            name: "waterlock-ldap-auth",
            connection: {
                url: "ldap://...",
                bindDn: "uid=myadminusername,ou=users,o=example.com",
                bindCredentials: "mypassword",
                searchBase: "...",
                searchFilter: "...",
                cache: true
            },
            attributes: {
                fullName: {
                    displayName: {
                        type: 'string'
                    }
                },
                email: {
                    userPrincipalName: {
                        type: 'string'
                    }
                },
                uid: {
                    userPrincipalName: {
                        type: 'string'
                    }
                }
            }
        }
]
```

When i run the sails lift command I get an error

```
MYPROJECT\api\models\ResetToken.js:11
  attributes: require('waterlock').models.resetToken.attributes({
                                                    ^
TypeError: Cannot read property 'attributes' of undefined
    at Object.<anonymous> (MYPROJECT\api\models\ResetToken.js:11:53)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at MYHOME\AppData\Roaming\npm\node_modules\sails\node_modules\include-all\index.js:129:29
    at Array.forEach (native)
    at requireAll (MYHOME\AppData\Roaming\npm\node_modules\sails\node_modules\include-all\index.js:44:9)
    at buildDictionary (MYPROJECT\node_modules\sails\node_modules\sails-build-dictionary\index.js:68:14)
    at Function.module.exports.optional (MYPROJECT\node_modules\sails\node_modules\sails-build-dictionary\index.js:160:9)
    at Hook.loadModels (MYPROJECT\node_modules\sails\lib\hooks\moduleloader\index.js:279:23)
    at Hook.bound [as loadModels] (MYPROJECT\node_modules\sails\node_modules\lodash\dist\lodash.js:729:21)
    at Array.async.auto.models (MYPROJECT\node_modules\sails\lib\hooks\orm\load-user-modules.js:23:23)
    at MYPROJECT\node_modules\sails\node_modules\async\lib\async.js:484:38
```

**Comment by fladi** [comment]
The [line](https://github.com/waterlock/waterlock-local-auth/blob/2897e12d240035b2b58ed114260bf82194192acf/lib/templates/models/ResetToken.js#L11) in your error indicates that you are using [waterlock-local-auth](https://github.com/waterlock/waterlock-local-auth). `waterlock-ldap-auth`should not install a `ResetToken` model.
Can you describe the way you set up `waterlock` and `waterlock-ldap-auth`?

**Comment by iit-federico-bozzini** [comment]
You are right. 

I installed waterlock and waterlock-local-auth and run the "generate all" command. After that I installed the waterlock-ldap-auth module.

The error came probably from the fact that I was using your module with the files generated for the local auth module. 

Maybe I just misunderstood the documentation about waterlock and thought that running "generate all" once and that adding multiple auth modules was perfectly fine. Instead it looks like different modules don't play nice together when using the "generate all" command and they overwrite each other files.
